users_app/tests.py

```diff
@@ -1,13 +1,19 @@
 from http import HTTPStatus
 from parameterized import parameterized
 
-from django.test import TestCase, Client
+from django.test import (
+    Client,
+    TestCase,
+)
 from django.contrib.auth import get_user_model
 from django.urls import reverse
 from social_django.models import UserSocialAuth
 
 from users_app.models import User
-
+from supplier_app.models import (
+    Company,
+    CompanyUserPermission,
+)
 SUPPLIER_HOME = '/suppliersite/supplier'
```

Comment:
As I commented previously, try to avoid the use of global variables, it's a very easy way to have bugs.